scalafix-tests/output/src/main/scala/test/NoExtendsApp.scala

```diff
@@ -0,0 +1,51 @@
+package test
+
+object Main {
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+trait Something
+object Main2 extends Something{
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+object Main3 extends Something{
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+object Main4 extends Something
+
+object Main5 extends Something {}
+
+object Main6
+  extends Something
+
+object Main7
+  extends Something
+
+object Main8
+  extends Something
+
+object Main9 extends
+  Something
+
+import scala.{ App => Ppa }
```

Comment:
minor thing, in these (hopefully rare) cases, this import becomes unused. I deemed fixing this in a single pass too daunting. Users may have to manually handle this or run the `NoUnusedImports` rewrite right after.